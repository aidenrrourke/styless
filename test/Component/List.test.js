import React from 'react';
import renderer from 'react-test-renderer';
import styled from 'styled-components';
import 'jest-styled-components';

test.skip('Returns the number of elements in a value list', () => {
    const Div = styled.div`
        @list: "banana", "tomato", "potato", "peach";
        n: length(@list);
        n2: length(1px solid #0080ff);
	`;
    expect(renderer.create(<Div/>).toJSON()).toMatchSnapshot();
});

test.skip('Returns the value at a specified position in a list.', () => {
    const Div = styled.div`
        @list: apple, pear, coconut, orange;
        value: extract(@list, 3);
	`;
    expect(renderer.create(<Div/>).toJSON()).toMatchSnapshot();
});