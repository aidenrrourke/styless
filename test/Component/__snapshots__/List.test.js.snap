// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Returns the number of elements in a value list 1`] = `
.c0 {
  n: 4;
  n2: 3;
}

<div
  className="c0"
/>
`;

exports[`Returns the value at a specified position in a list. 1`] = `
.c0 {
  value: coconut;
}

<div
  className="c0"
/>
`;
